pkg/data/domain/app/validate.go

```diff
@@ -0,0 +1,418 @@
+package app
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"path"
+
+	applicationv1alpha1 "github.com/giantswarm/apiextensions/v2/pkg/apis/application/v1alpha1"
+	"github.com/xeipuuv/gojsonschema"
+	"gopkg.in/yaml.v3"
```

Comment:
FMT states that we should use `sigs.k8s.io/yaml` instead.

https://github.com/giantswarm/fmt/blob/master/go/yaml_marshalling.md